lib/bundling.js

```diff
@@ -60,12 +60,18 @@ function noop() {
  */
 function computeBundleId(obj, discriminatorFields) {
   var ids = [];
+  var hasIds = false;
   for (var i = 0; i < discriminatorFields.length; ++i) {
     var id = _.at(obj, discriminatorFields[i])[0];
     if (id === undefined) {
-      return undefined;
+      ids.push('');
```

Comment:
This is used only within this file to distinguish tasks. Looking weird is okay, more serious problem could happen if it can conflict with other type of data.